packages/client/src/components/Header.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+function logout() {
+	console.log('logout');
+}
+</script>
+
+<template>
+	<nav id="navbar">
```

Comment:
No need for ID, just use the type selector

Reply:
@YohannParis why? I agree that may not be needed for all but some may be useful for e2e testing

Reply:
True, but then we can add the ID if needed, not in case of its needed.